kolibri_explore_plugin/assets/src/views/CustomChannelPresentationApp.vue

```diff
@@ -1,12 +1,21 @@
 <template>
 
-  <iframe
-    ref="iframe"
-    class="custom-presentation-iframe"
-    frameBorder="0"
-    :src="rooturl"
-  >
-  </iframe>
+  <div>
```

Comment:
Hmm but you are adding a div wrapper anyways.

Reply:
Yes, it's the only way here we've just the iframe so we should wrap or add the overlay in the parent component, that'll be a bit more complex because we should pass a callback property or use the store to show/hide.

In any case, I'll try to find a clean solution for that so there's no wrap on this component.

Reply:
I've moved the overlay to the parent element and manage the state with signals.